backend/src/idl/policy.proto

```diff
@@ -0,0 +1,57 @@
+syntax = "proto2";
+package fk.prof.backend.proto;
+option java_outer_classname = "PolicyDTO";
+
+message PolicyDetails {
+    required Policy policy = 1;
+    required string modified_by = 2;
+    required string modified_at = 3;
+    required string created_at = 4;
+    required string last_scheduled = 5;
```

Comment:
lets remove this? We are not going to update policy every time some work is scheduled. Can have this once we add /status API